### DISCUSSION TITLE
Beacon Events will be too hard to mod

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14808] [Unstable feedback]
If you want to mod scripted events for beacons like Shockjock2, you'll have to override the entire eventset to insert your own events along shockjock2
This makes mods that add beacon events incompatible, and adding beacon events harder than it should be

![image](https://github.com/user-attachments/assets/8ae99ecd-1f0b-4428-88fd-b3d8c19acb70)

The eventset should trigger an event TAG, so the "empty" eventset triggers one scriptedevent tagged "beaconscriptedevent", and then modders can just add their own events with commonness using the tag

**Comment by girlyguppy** [comment]
we need this real
